src/content/tools/devtools/network.md

```diff
@@ -55,6 +55,20 @@ my-endpoint m:get t:json s:200
 https s:404
 ```
 
+### Recording network requests on app startup
+
+To record network traffic on app startup, you can start your app in a paused state then, begin recording before your app starts:
+
+1. Start your app in a paused state:
+    * `flutter run`
+      * use the `--start-paused` flag.
+    * `dart run`
+        * use the `--pause-isolates-on-start` flag.
```

Comment:
can you do a full command here for both of these?. I've had trouble getting the dart flag to work before because I think the flag has to be in the right spot